pkg/redfish/rfcomponents.go

```diff
@@ -1830,6 +1846,51 @@ func (s *EpSystem) discoverComponentEPEthInterfaces() {
 			}
 		}
 	}
+
+	if foxconnMACWorkarround {
+		var mgr *EpManager = nil
+		var ok bool = false
+		// Get the first manager linked to in the system object
+		for _, oid := range s.ManagedBy {
+			mgr, ok = s.epRF.Managers.OIDs[oid.Basename()]
+			if ok {
+				break
+			}
+		}
+		// If no link to ManagedBy Manager in system object, just pick
+		// the first manager (there is likely only one)
+		if !ok {
+			for _, m := range s.epRF.Managers.OIDs {
+				mgr = m
+				break
+			}
+		}
+		if mgr != nil {
+			foundMac := false
+			for _, eth := range mgr.ENetInterfaces.OIDs {
+				if eth.EtherIfaceRF.Id == "eth0" {
+					bmcMac := eth.EtherIfaceRF.MACAddress
+					mac, err := GetOffsetMACString(bmcMac, int64(ParadiseMacWarInc))
+					if err != nil {
+						errlog.Printf("%s: failed to increment the bmc mac %s by %d, error: %s",
+							s.ID, bmcMac, ParadiseMacWarInc, err)
+						continue
```

Comment:
Should this 'continue' be a 'break' since no other interface will match "eth0"?  Not a big deal either way, both will turn out to be functionally the same.

Reply:
Changing to break is ok.

Reply:
I missed this when I was testing earlier.  Since they are both functionally the same, I will leave it with continue.